packages/cli/tests/integration/rpc.appmap.spec.ts

```diff
@@ -12,6 +12,31 @@ describe('RPC', () => {
   afterEach(async () => await rpcTest.teardownEach());
   afterAll(async () => await rpcTest.teardownAll());
 
+  describe('appmap.stats', () => {
+    it('can be retrieved', async () => {
```

Comment:
@dividedmind one weird thing - this test case should be showing up in the AppMap CI report as "New AppMap", but it's not.

Reply:
The main branch is also using a different revision of the appmap.yml though so, I don't know what the impact of that is. Still, I'd expect to see changes in the PR report.

Reply:
Hmm. Did you check if an AppMap actually gets created?